.travis.yml

```diff
@@ -22,7 +26,7 @@ env:
 matrix:
   include:
     - os: linux
-      env: TOOLSET=gcc COMPILER=g++-7 CXXSTD=c++11
+      env: TOOLSET=gcc COMPILER=g++-7 CXXSTD=c++17
```

Comment:
But here we should test it with gcc and clang and also different c++ standards right?

Reply:
Yes, exactly. (at least) one with c++11 and one with c++17, as a start. We can add other compilers later. Then, in the tensor test Jamfile, we can conditionalize the tests on that. (In fact, we should use the predefined property `cxxstd` for that, as that makes it easy to express the logic of "only do this when `cxxstd=17`, etc.